src/code_80005FD0.c

```diff
@@ -1628,10 +1628,10 @@ void func_800097E0(void) {
 
         switch(gCurrentCourseId) {
             case COURSE_KALAMARI_DESERT:
-                func_80012AC0();
+                update_vehicle_trains();
```

Comment:
I'm considering `update_trains()`

I can see including vehicle being good. But I dunno.

Reply:
is just to say clearly what type it is like update_actor_... he can be good to also do that for object I will do it